iopipe/config.py

```diff
@@ -14,6 +15,7 @@ def set_config(**config):
     config.setdefault('network_timeout', 5000)
     config.setdefault('timeout_window', os.getenv('IOPIPE_TIMEOUT_WINDOW', 150))
     config.setdefault('install_method', 'manual')
+    config.setdefault('enabled', is_enabled())
```

Comment:
@mislavcimpersak Couldn't this just be:

```python
config.setdefault('enabled', strtobool(os.getenv('IOPIPE_ENABLED', 'true')))
```

you could make the docstring inline for this setting. I plan to add inline docstrings for the others as well.

Reply:
Sure, it could be a one-liner. But this way it's easier to write tests for it and thus easier to maintain and debug if necessary.

Reply:
Plus, #97 could also benefit from `is_enabled` method since in it a check if platform is Linux could be done an just simply disabling IOpipe.